roles/openshift_master/templates/docker-cluster/atomic-openshift-master-controllers.service.j2

```diff
@@ -22,12 +22,39 @@ ExecStart=/usr/bin/docker run --rm --privileged --net=host \
   {% if l_bind_docker_reg_auth | default(False) %} -v {{ oreg_auth_credentials_path }}:/root/.docker:ro{% endif %}\
   {{ osm_image }}:${IMAGE_VERSION} start master controllers \
   --config=${CONFIG_FILE} $OPTIONS
+ExecStartPre=-/usr/bin/docker rm -f {{ openshift_service_type}}-master-kube-controllers
```

Comment:
@mrunalp @runcom doesn't docker get really angry when you use `rm -f` ?  Or is docker ok with this?

Reply:
That was usually related to some devmapper stuff, I doubt we have that issue anymore but yeah, Docker at least had an issue with rm -f